### DISCUSSION TITLE
既然都这么有模有样了……

### DISCUSSION THREAD

**Discussion by yzysrewrwqr** [share121/inter-knot#3062] [其他]
要不，开发一个板块，完全按游戏内人设来回复<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by lllliufeng** [comment]
什么语c
